bus-mapping/src/circuit_input_builder.rs

```diff
@@ -572,4 +594,26 @@ impl<P: JsonRpcClient> BuilderClient<P> {
         )?;
         Ok((builder, eth_block))
     }
+
+    /// Check if a tx is invalid or not.
+    pub fn check_invalid_txs(
+        &self,
+        block: &eth_types::Block<eth_types::Transaction>,
+        geth_traces: &mut [eth_types::GethExecTrace],
+        txs_rlp: &eth_types::Bytes,
+    ) {
+        let txs:Vec<eth_types::Transaction> = Rlp::new(txs_rlp).as_list().expect("invalid txs rlp");
+        for (tx, geth_trace) in block.transactions.iter().zip(geth_traces.iter_mut()) {
+            let is_anchor = taiko_anchor::is_anchor(tx);
+            if is_anchor {
+                geth_trace.invalid = Some(false);
```

Comment:
I think we can not assume anchor is always valid, although we will have a circuit for anchor later.

Reply:
Removed is_anchor.